### PR TITLE
Marques johnson

### PR DESCRIPTION
**Pull request by MarquesJ8023**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -40,3 +40,72 @@ const siteContent = {
...
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+const navA = document.querySelectorAll('a');
+console.log(navA);
+
+
+navA.forEach( function( currentValue ) {
+  currentValue.style.color = "green";
+});
+navA[0].textContent = 'Services';
```
Comment (ebcitron) [inline js/index.js:51]: Next time try looping through with an If statement or For Each

File: js/index.js [hunk 2]
```diff
@@ -40,3 +40,72 @@ const siteContent = {
...
+console.log(textContent);
+console.log(features);
+
+textContent[0].querySelector('h4').textContent = "Features";
+textContent[1].querySelector('h4').textContent = "About";
+textContent[2].querySelector('h4').textContent = "Services";
+textContent[3].querySelector('h4').textContent = "Product";
+textContent[4].querySelector('h4').textContent = "Vision";
+
+textContent[0].querySelector('p').textContent = "Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.";
```
Comment (ebcitron) [inline js/index.js:94]: Im curious why you did it like this. If you ever read this, bring it up

**Review by ebcitron — APPROVED** [review]
Overall great job man! Keep it up

**Comment by MarquesJ8023** [comment]
Okay.

**Pull request merged**